### DISCUSSION TITLE
make menu buttons for command looks better

### DISCUSSION THREAD

**Discussion by khanfar** [n3d1117/chatgpt-telegram-bot#221] [Ideas]
hi , im just have this code to make 4 buttons menu for /help  /reset /stats / resend  ...  can you look at it and add in next update https://github.com/khanfar/4-buttons-telegram-bot-menue.git

**Comment by Shady-Dev** [comment]
You should remove your bot API token from your code page

**Comment by n3d1117** [comment]
That's cool @khanfar, could you open a PR?

**Reply by khanfar** [reply]
What you mean by pr ?

**Reply by n3d1117** [reply]
Pull request

**Reply by khanfar** [reply]
Give me allow to pull
